From 479b46d2ed42b576cf2aee2ecbd76de7940d0cfe Mon Sep 17 00:00:00 2001
From: Yong Li <yong.b.li@linux.intel.com>
Date: Fri, 10 Aug 2018 16:23:13 +0800
Subject: [PATCH] Increase the default watchdog timeout value

The default timeout for poweron is 30 seconds,
but currently the host power on needs 120+ seconds
due to unimplemented ipmi commands for BIOS.

Increase the value as a workaround,
to avoid the watchdog timeout during power on.
Will adjust this value in the future

Signed-off-by: Yong Li <yong.b.li@linux.intel.com>
---
 yaml/xyz/openbmc_project/State/Watchdog.interface.yaml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/yaml/xyz/openbmc_project/State/Watchdog.interface.yaml b/yaml/xyz/openbmc_project/State/Watchdog.interface.yaml
index f54ec3b6e4fb..ede961da1942 100644
--- a/yaml/xyz/openbmc_project/State/Watchdog.interface.yaml
+++ b/yaml/xyz/openbmc_project/State/Watchdog.interface.yaml
@@ -37,7 +37,7 @@ properties:
       type: uint64
       description: >
           Time interval to arm the watchdog, in milli-second.
-      default: 30000
+      default: 600000
     - name: TimeRemaining
       type: uint64
       description: >
-- 
2.17.1

